src/mds/MDSRank.cc

```diff
@@ -3828,6 +3853,10 @@ const char** MDSRankDispatcher::get_tracked_conf_keys() const
     "mds_extraordinary_events_dump_interval",
     "mds_inject_rename_corrupt_dentry_first",
     "mds_inject_journal_corrupt_dentry_first",
+    "mds_dmclock_enable",
+    "mds_dmclock_reservation",
+    "mds_dmclock_weight",
+    "mds_dmclock_limit",
```

Comment:
why is this added twice, maybe merge error?

Reply:
There were merge issues. I will fix it.